internal/install/static_kibana_config_yml.go

```diff
@@ -17,7 +17,8 @@ xpack.fleet.enabled: true
 xpack.fleet.registryUrl: "http://package-registry:8080"
 xpack.fleet.agents.enabled: true
 xpack.fleet.agents.elasticsearch.host: "http://elasticsearch:9200"
-xpack.fleet.agents.kibana.host: "http://kibana:5601"
+xpack.fleet.agents.fleetServerEnabled: true
+xpack.fleet.agents.kibana.host: "http://localhost:8220"
```

Comment:
This will change as part of https://github.com/elastic/beats/pull/24713 and https://github.com/elastic/kibana/pull/94364

Reply:
Thank you for reviewing this. What is the recommendation though? Should I wait until these PRs are merged?

Reply:
I would like to switch over elastic-package to fleet-server as soon as possible to have early testing. At the same time I do not want to impact the integrations team with potential bugs / changes. The above changes can only land if a dependency from endpoint is also merged at the same time. I think an easy trick on our side will be to just ahve both config options in already so things will keep working. @nchaulet is this assumption correct?

Reply:
Yes thats correct we can have both options as soon my PR for fleet server hosts is merged (hopefully soon :) ) https://github.com/elastic/kibana/pull/94364